packages/cli/src/commands/releasegold/withdraw.test.ts

```diff
@@ -31,10 +32,13 @@ testWithGanache('releasegold:withdraw cmd', (web3: Web3) => {
     await timeTravel(100000000, web3)
     const releaseGoldWrapper = new ReleaseGoldWrapper(kit, newReleaseGold(web3, contractAddress))
     const beneficiary = await releaseGoldWrapper.getBeneficiary()
-    const balanceBefore = await kit.getTotalBalance(beneficiary)
-    await Withdraw.run(['--contract', contractAddress, '--value', '10000000000000000000000'])
-    const balanceAfter = await kit.getTotalBalance(beneficiary)
-    expect(balanceBefore.CELO!.toNumber()).toBeLessThan(balanceAfter.CELO!.toNumber())
+    const balanceBefore = await (await kit.getTotalBalance(beneficiary)).CELO!
+    // Use a value which would lose precision if converted to a normal javascript number
+    const withdrawalAmount = '10000000000000000000005'
+    await Withdraw.run(['--contract', contractAddress, '--value', withdrawalAmount])
+    const balanceAfter = await (await kit.getTotalBalance(beneficiary)).CELO!
```

Comment:
nit/?: here and above, is the `await` (in `await ().CELO!`) necessary?

Reply:
Good catch!  What's going on here is I was relying on VS Code here when I add `.CELO!`, and I guess what happens in detail is that it adds the parentheses and inner await, while leaving the original await as is when it can be removed.  Not the first time I've seen it, to be honest, but I forgot to check for it afterwards.